package.json

```diff
@@ -15,6 +15,7 @@
     "bootstrap": "^4.4.1",
     "compression": "^1.7.4",
     "express": "^4.17.1",
+    "nodemon": "^2.0.3",
```

Comment:
This is not needed i guess. You can directly run `npm run dev`. Moreover this should be a dev dependency and not dependency :)

Reply:
@beccabryn